### DISCUSSION TITLE
Issue with directory scanning on Synology NAS

### DISCUSSION THREAD

**Discussion by chemage** [LMS-Community/slimserver#1057] [Q&A] [answered]
Hello.

I've just updated to LMS 8.5.1 and the issue remains. I have a directory structure with lots of folders and MP3 files.
With previous version LMS 8.5.0 it was the same.
Only a few of the MP3's get discovered by the scan.
The discovered items are not all in the same subfolders.
The other subfolders get ignored or the files are not compatible for some reason.
But with about 15k MP3s, I find it hard to believe that it would find only a few.

Are there any path limitations ? 
Or something else I missed ?

Should I open an issue ?
Does someone else have the same issue ?
I can provide versions, configs, logs, etc.

Cheers.
Marcel

**Comment by michaelherger** [comment, marked as answer]
Hi Marcel

I'd recommend you join the community at http://forums.slimdevices.com - you'll find a ton of helpful people there. Please post information about your setup: are you running LMS in Docker? The scanner.log.zip (which you can find in Settings/Information) etc. Most likely this is a permissions issue, and there are people who have a similar or identical setup.

**Comment by chemage** [comment]
Hello Michael.

Sorry for not replying earlier.
I have found a few resources which will help with permissions (I figured I never changed any permissions, so it shouldn't be an issue, but somehow, it was different on some folder in File Station).
I haven't figured out how to set them up properly, as someone stated out, that the user/groups in the container are not the ones in DSM.
I will try set up local users in both, with the same ID, to see how that goes. 
For the time being, I set Everyone:RO on the music folder, which works fine, but is less secure.
Thanks a lot for continuing developing this great solution.
Cheers.
Marcel

**Reply by michaelherger** [reply]
Don't mess too much with permissions. You're likely going to cause even more issues... Really: ask in the forums, and I'm sure you'll get help.

**Comment by chemage** [comment]
For reference: https://www.synoforum.com/threads/docker-permissions-how-and-where.5022/
